test/battle/ability/seed_sower.c

```diff
@@ -29,7 +29,7 @@ DOUBLE_BATTLE_TEST("Multi-target moves hit correct battlers after Seed Sower is
 {
     u32 j, k, l;
     u16 usedMove = MOVE_NONE;
-    static const u16 moves[] = {MOVE_HYPER_VOICE, MOVE_SURF};
+    static const u16 moves[] = { MOVE_HYPER_VOICE, MOVE_SURF};
```

Comment:
Here too, no need for this space in arrays. Maybe check the entire PR? I'll continue reviewing after